tailcfg/tailcfg.go

```diff
@@ -2253,6 +2253,9 @@ const (
 
 	// NodeAttrAutoExitNode permits the automatic exit nodes feature.
 	NodeAttrAutoExitNode NodeCapability = "auto-exit-node"
+
+	// NodeAttrStoreAppCRoutes enables storing app connector routes persistently.
```

Comment:
We have some node attrs that do something immediately and some that unlock a capability, permitting it to be used if desired. We try to distinguish which are which.

From the code, it sounds like this being present immediately changes the behavior, as opposed to providing the option for it to be configured, right?

If so, I'd say something like "configures the node to store app connector routes persistently" or "makes" or "instructs".

Reply:
Addressed in fixup commit. 👍

Reply:
How do you use fixup commits? (there are several workflows, none of which GitHub seems to support well) You squash this all down into one commit before merging?

Reply:
I was planning on seeing if that addressed your concern and then squashing that fixup commit into the commit it targets, so I'm planning on this PR having 5 commits in it (if that all looks good to you).

So in my head we are currently in the "show Brad the specific diffs in response to his comments" phase, after which there's a "squash the fixups in to the target commits" phase.

Am I answering the right question here?

Reply:
Oh, sure. I wasn't expecting that use, but that's fine. I can also see the diffs between versions without fixup commits.